visvis2/renderers/_wgpu.py

```diff
@@ -95,7 +121,7 @@ def compose_pipeline(self, wobject):
             ctypes.memmove(buffer.mapping, array.ctypes.data, nbytes)
             buffer.unmap()
             shader_location = len(buffers)
-            buffers[shader_location] = buffer
+            # buffers[shader_location] = buffer
             vbo_des = {
                 "array_stride": 3 * 4,
```

Comment:
is this 3 * 4 because every vertex has 3 x float32 (4 bytes)?

Reply:
Exactly